### PR TITLE
magento first draft

### PR DESCRIPTION
**Pull request by codeallthethingz**

### CODE REVIEW

File: roles/engineering-magento-developer.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Engineering - Magento Developer
```
Comment (leukanos) [inline roles/engineering-magento-developer.md:1]:
```suggestion
# Engineering - Magento 2 Developer
```

File: roles/engineering-magento-developer.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+<welcome.snippet/>
+
+## What it's like to be a Developer
+As a developer, you spend the first few minutes of the day, reviewing your inbox, asana tasks, outstanding code reviews to make sure you’re not blocking anyone on your team.  As our magento specialist you also have a bunch of questions and requests from others in the company about upgrading magento, testing environments for magento and other requests about that platform.
+
+You maintain our magento environments, adding e2e tests where needed, and training other people how to use those environments.
```
Comment (ktruszkowska) [inline roles/engineering-magento-developer.md:7]: e2e tests = MFTF?

File: roles/engineering-magento-developer.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+You love learning new technologies and solving problems in novel ways.  You enjoy creating software with a team of peers.
+
+## Why we might like you
+You enjoy working with others and helping mentor developers at all skill levels.  You are a strong communicator who takes the time to interact with people across different disciplines. You have a keen sense of community and are always looking for new ways to expand the work of others.
+
+<what-you-need.snippet/>
+
+## Skills that are important to this role
+
+<skills>
```
Comment (ktruszkowska) [inline roles/engineering-magento-developer.md:27]: Elasticsearch wouldn't be nice to have as well?

File: roles/engineering-magento-developer.md (hunk 2 above)
Comment (codeallthethingz) [inline roles/engineering-magento-developer.md:7]:
```suggestion
You maintain our magento environments, adding e2e (MFTF) tests where needed, and training other people how to use those environments.
```

File: roles/engineering-magento-developer.md (hunk 3 above)
Comment (codeallthethingz) [reply roles/engineering-magento-developer.md:27]: Nice to have, but it's a big list already. :)

**Pull request merged**